src/main/java/lotto/Lotto.java

```diff
@@ -0,0 +1,49 @@
+package lotto;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import static java.util.Collections.*;
+
+public class Lotto {
+
+    private static final String DELIMITER = ",";
+    private final List<Integer> lottoNumbers;
+
+    public Lotto(String textNumbers) {
+        this(Arrays.stream(textNumbers.split(DELIMITER))
+                .map(String::trim)
+                .map(Integer::parseInt)
+                .collect(Collectors.toList()));
+    }
+
+    public Lotto(List<Integer> lottoNumbers) {
+        this.lottoNumbers = lottoNumbers;
+    }
+
+    public int countMatchingNumbers(List<Integer> otherLottoNumbers) {
+        return (int) otherLottoNumbers.stream()
+                .filter(lottoNumbers::contains)
+                .count();
+    }
```

Comment:
```suggestion
    public int countMatchingNumbers(Lotto otherLottoNumbers) {
        return (int) otherLottoNumbers.lottoNumbers
                .stream()
                .filter(lottoNumbers::contains)
                .count();
    }
```
객체를 서로 호출하도록하면 같은 `Lotto`객체이므로 외부에서 getter를 따로 호출하여 선언할 필요가 없을 것 같아요 😄